src/index.html

```diff
@@ -6,9 +6,58 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home"><img src="images/logo.png" alt="logo"></a>
+      <nav class="nav__item">
+        <ul class="nav__list">
+          <li>
+            <a href="#apple" class="is-active nav__link">
+                Apple
+            </a>
+          </li>
+          <li>
+            <a href="#samsung" class="nav__link">
+                Samsung
+            </a>
```

Comment:
```suggestion
            <a href="#samsung" class="nav__link">
              Samsung
            </a>
```
Consider fixing indents according to the code style guide